test/common_utils.py

```diff
@@ -16,3 +16,11 @@ def create_temp_assets_dir():
     copytree(os.path.join(TEST_DIR_PATH, "assets"),
              os.path.join(tmp_dir.name, "assets"))
     return tmp_dir.name, tmp_dir
+
+
+def _num_stft_bins(signal_len, fft_len, hop_length, pad):
+    return (signal_len + 2 * pad - fft_len + hop_length) // hop_length
+
+
+def _approx_all_equal(x, y, atol=1e-7):
```

Comment:
would torch.allclose be better here?
https://pytorch.org/docs/stable/torch.html#torch.allclose

Reply:
Absolutely. Strange that I missed it :o